operations/jsonnet-compiled/util/example/main.jsonnet

```diff
@@ -5,10 +5,10 @@ local tempo = import 'microservices/tempo.libsonnet';
 tempo {
   _images+:: {
     tempo: 'grafana/tempo:latest',
-    tempo_query: 'grafana/tempo-query:latest',
```

Comment:
Maybe comment instead of delete?

Reply:
will keep it as is, this var is no-op when `tempo_query.enabled: false`